src/keyring/KeyringController.ts

```diff
@@ -380,6 +384,11 @@ export class KeyringController extends BaseController<
     return this.fullUpdate();
   }
 
+  /**
```

Comment:
I am not certain but do you mind filing a new issue with this question? That way we can start a discussion about it there.

Reply:
https://github.com/MetaMask/controllers/issues/801